src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -398,12 +398,10 @@ private static String replaceUnpredictableValuesWithPlaceholders(String content)
                       .replace(TimeHelper.formatDate(now), "${today}")
                       // date/time now e.g [Thu, 07 May 2015, 07:52 PM] or [Thu, 07 May 2015, 07:52 PM UTC]
                       .replaceAll(dateTimeNow + REGEX_DISPLAY_TIME, "\\${datetime\\.now}")
-                      // jQuery files
-                      .replace("/js/lib/jquery.min.js", "${lib.path}/jquery.min.js")
-                      .replace("https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js", "${lib.path}/jquery.min.js")
-                      // jQuery-ui files
-                      .replace("/js/lib/jquery-ui.min.js", "${lib.path}/jquery-ui.min.js")
-                      .replace("https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js", "${lib.path}/jquery-ui.min.js")
+                      // jQuery js file
+                      .replace(Const.SystemParams.getjQueryFilePath(TP.isDevServer()), "${lib.path}/jquery.min.js")
```

Comment:
No diff. `TestProperties.isDevServer()` has been there since forever. If you're aiming for uniformity, I'm the one who should make the change, i.e `Config.isDevServer()`. Is that what you mean?